.github/workflows/pr-ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python: [ 3.7, 3.8, 3.9 ]
+        python: [ 3.8, 3.9, "3.10", "3.11" ]
```

Comment:
is there a reason 3.10 and 3.11 are in quotes?

Reply:
I will switch them all to double quotes.  For some reason when we started adding two digits behind the `.` we needed to add quotes.

Reply:
Added in double quotes to all values since it is required in 3.10